### DISCUSSION TITLE
[Bug] SWINIR Denoising train error: No such file or directory:'data/DFWB/meta_info_DFWB8550sub_GT.txt'

### DISCUSSION THREAD

**Discussion by fnwkqwan** [open-mmlab/mmediting#1964] [Datasets] [answered]
### Prerequisite

- [X] I have searched [Issues](https://github.com/open-mmlab/mmagic/issues) and [Discussions](https://github.com/open-mmlab/mmagic/discussions) but cannot get the expected help.
- [X] I have read the [FAQ documentation](https://mmagic.readthedocs.io/en/latest/faq.html) but cannot get the expected help.
- [X] The bug has not been fixed in the [latest version (main)](https://github.com/open-mmlab/mmagic) or [latest version (0.x)](https://github.com/open-mmlab/mmagic/tree/0.x).

### Task

I'm using the official example scripts/configs for the officially supported tasks/models/datasets.

### Branch

main branch https://github.com/open-mmlab/mmagic

### Environment

TorchVision:0.12.0+cu113
OpenCV:4.8.0
MMEngine:0.8.1
MMCV:2.0.0rc4
MMCV Compiler: MSVC 192829924
MMCV CUDA Compiler:11.3
MMagic: 1.0.1+b058b41

### Reproduces the problem - code sample

python tools/train.py configs/swinir/swinir_s128w8d6e180_8xb1-lr2e-4-1600k_dfwb-grayDN15.py

### Reproduces the problem - command or script

/

### Reproduces the problem - error message

FileNotFoundError: No such file or directory:'data/DFWB/meta_info_DFWB8550sub_GT.txt'

### Additional information

I have prepared datasets by tools/dataset_converters/denoising 'Preparing Denoising Dataset' as The folder structure  looks like:

mmagic
├── mmagic
├── tools
├── configs
├── data
|   ├── denoising_gaussian_test
|   |   ├── Set12
|   |   ├── BSD68
|   |   ├── CBSD68
|   |   ├── Kodak
|   |   ├── McMaster
|   |   ├── Urban100
But there is no DFWB dataset mentioned in the configuration file, how can I get it?

**Comment by zengyh1900** [comment, marked as answer]
hi @fnwkqwan ,

`DFWB` denotes the combination of [DIV2K](https://cv.snu.ac.kr/research/EDSR/DIV2K.tar) (800 training images) + [Flickr2K](https://cv.snu.ac.kr/research/EDSR/Flickr2K.tar) (2650 images) + [WED](http://ivc.uwaterloo.ca/database/WaterlooExploration/exploration_database_and_code.rar)(4744 images) + [BSD500](http://www.eecs.berkeley.edu/Research/Projects/CS/vision/grouping/BSR/BSR_bsds500.tgz) (400 training&testing images).

These four datasets are usually combined and used as the training data of **color/grayscale image denoising** and **greyscale/color JPEG compression artifact reduction**. You can simply download and put them together and generate the annotation file so that the DataLoader can sample images from these four datasets during training.
